backend/tournesol/models/entity.py

```diff
@@ -163,6 +164,37 @@ def link_to_youtube(self):
             '<a href="https://youtu.be/{}" target="_blank">Play ▶</a>', self.video_id
         )
 
+    def criteria_scores_distributions(self, poll):
+        """Returns the distribution of critera score for the entities"""
+        min_score_base = -1.0
+        max_score_base = 1.0
+
+        # Fetch data with QuerySet
+        contributor_rating_criteria_score_list = [
+            [y for y in x.criteria_scores.all()]
+            for x in self.contributorvideoratings.filter(poll=poll).prefetch_related(
+                "criteria_scores"
+            )
+        ]
```

Comment:
The below avoids using variables `x` and `y`
```suggestion
        contributor_rating_criteria_score_list = [
            list(contributor_rating.criteria_scores.all())
            for contributor_rating in self.contributorvideoratings.filter(poll=poll).prefetch_related(
                "criteria_scores"
            )
        ]
```